bigquery/indexer.py

```diff
@@ -159,25 +123,27 @@ def _sample_scripts_by_id_from_export(
                     row[has_name] = True
                 else:
                     row[has_name] = False
-
-        script = UPDATE_SAMPLES_SCRIPT % (sample_id_column, sample_id_column)
-        yield participant_id, {
-            'source': script,
-            'lang': 'painless',
-            'params': {
-                'sample': row
-            }
-        }
+        if participant_id in all_table_documents and 'samples' in all_table_documents[
+                participant_id]:
+            all_table_documents[participant_id]['samples'][0].update(row)
```

Comment:
I tried encode and noticed that each participant doc only has one sample doc. You need to append to samples list here, instead of overwrite first item in the list.